### PR TITLE
Nginx

### PR DESCRIPTION
**Pull request by wangxiaoming9168**

### CODE REVIEW

File: 4.2/Dockerfile [hunk 1]
```diff
@@ -146,4 +146,4 @@ EXPOSE 8080
 
 COPY conf.d/env $PROJECT_DIR/.env
 
-CMD ["/app/start.sh"]
+CMD ["/app/start.sh"]
```
Comment (philipkozeny) [inline 4.2/Dockerfile:149]: make sure that there is always a newline at the end of the file

File: 4.2/cmd
Comment (philipkozeny) [inline 4.2/cmd:1]: this file can be removed altogether

**Review by philipkozeny — COMMENTED** [review]
Thanks for the PR! Here's a quick review without testing.

**Comment by leonardehrenfried** [comment]
The latest version of the image doesn't use a separate web server anymore, so neither apache nor nginx are necessary.

**Pull request closed without merging**